src/python-common/ceph/deployment/translate.py

```diff
@@ -116,73 +116,53 @@ def run(self):
                 raise ValueError('Number of data devices must match number of '
                                  'wal devices for raw mode osds')
 
-        # For this use case we don't apply any custom crush_device_classes
-        # Note that filestore is not supported anymore by the DriveGroupSpec
-        if self.spec.objectstore == 'filestore':
-            # for lvm batch we can just do all devices in one command
-            devs: List = sum(list(devices.values()), [])
-
-            cmd = "lvm batch --no-auto"
-
-            cmd += " {}".format(" ".join(devs))
-
-            if self.spec.journal_size:
-                cmd += " --journal-size {}".format(self.spec.journal_size)
-
-            if journal_devices:
```

Comment:
```
ceph/deployment/translate.py:98:9: F841 local variable 'journal_devices' is assigned to but never used
```

Reply:
Sorry, missed that line: updated in the current patchset.